README.md

```diff
@@ -2,107 +2,18 @@
 
 here lies the code that runs Odin's right hand bot... 
 
-## Setup
+This bot is used in a server for [theodinproject.com](https://www.theodinproject.com), an open source web development bootcamp, and is used by tens of thousands of Odinites. This bot has been built with NodeJS and [DiscordJS](https://discord.js.org/#/docs/main/stable/general/welcome).
 
-First, fork this repository, then `git clone` the repository. Then, `cd` into the folder and run `npm install` to install all dependencies. Once the repository has been cloned and the dependencies have been installed, you will need to create a `.env` file using `touch .env` or your editor's filesystem, and update it with your own Discord API key. A sample of this file can be found at `.env.sample`, including formatting for your key that will be used when you run the bot. You may skip to the [**Commands**](https://github.com/TheOdinProject/odin-bot-v2/blob/master/README.md#commands) section if you do not want to run your own instance of the Odin-Bot.
+This repo contains the main app for the Odin Project which can be found online at [theodinproject.com](https://www.theodinproject.com). It pulls in lesson and project content from our [curriculum repo](https://github.com/TheOdinProject/curriculum).
 
-### Getting Your Discord API Key
+Follow [this link](https://discord.gg/fbFCkYabZB) to join our Discord server!
 
-1. To get your key, you will need to go to https://discord.com/developers/applications
-2. Create a new application by clicking on the "New Application" button on the top right corner of the screen. 
-3. Give your application a name. Try to make the name unique, as your bot will share this name by default. The name may be changed after the bot is created, but if Discord has too many users with the same name as your application, it will not allow you to create a bot until you use a more unique application name.
-4. After creating the application, you will then need to add a bot by clicking the "Bot" tab on the left-hand sidebar, and pressing "Add Bot". You may change the bot's default username that will be displayed in your server at this point.
-5. Click the "Copy" button to copy your bot's token, then paste it into the `.env` file as the value for `DISCORD_API_KEY`. 
-**Note: This is sensitive information, ensure the `.env` file is listed in your .gitignore and NEVER push this key to Github!**
-6. Return to the Applications page and click on your bot, then navigate to the OAuth2 tab.
-7. Under Scopes, check off "bot".
-8. Under the Bot Permissions you may give your new instance of Odin-Bot as many or as few permissions as you'd like. At minimum, it should have permission to send messages, manage messages, embed links, and view channels.
-9. Paste the resulting URL into your browser and then invite the bot to your server of choice!
+## Getting Started
```

Comment:
What do you think about combining this with the contributing section below? 

They seem to go hand in hand, but I could definitely be overlooking something.

Maybe something like:

---
### Contributing
Thank you for your interesting contributing to the Odin Bot. The Odin Project depends on open source contributions to grow, improve, and thrive. We welcome contributions from beginners and experienced developers alike.

To find out more about how you can contribute, please read our [getting started](https://github.com/TheOdinProject/odin-bot-v2/wiki) and  [contributing guides](https://github.com/TheOdinProject/odin-bot-v2/wiki/Contributing-Guide).